openreview/conference/builder.py

```diff
@@ -188,6 +190,21 @@ def set_homepage_layout(self, layout):
     def set_double_blind(self, double_blind):
         self.double_blind = double_blind
 
+    def set_submission_public(self, submission_public):
+        self.submission_public = submission_public
+
+    def get_submission_readers(self):
+        return [
+            self.get_area_chairs_id(),
+            self.get_reviewers_id()
+        ]
```

Comment:
Why doesn't this include PCs?

Reply:
We are including the conference group id in all the places. Given that PCs are members of conference group, I thought that wasn't necessary. But I can change it.

Reply:
A note to that effect somewhere would be useful, especially if we remove PCs from the conference group for some reason down the line.

Reply:
In that case, I will add the PCs here so it is more clear.

Reply:
ok, I added it.